src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Ball ball1 = Lottery.getRandomBall();
+
+        Ball ball2 = Lottery.getRandomBall();
+
+        Ball ball3 = Lottery.getRandomBall();
+
+        System.out.println(ball1);
+
+        System.out.println(ball2);
+
+        System.out.println(ball3);
```

Comment:
use for loop to create three balls

Reply:
[loop for objects creating used with printing incl](https://github.com/mate-academy/jv-lottery/pull/366/commits/9c9a1e57b21bbd9704f0b300f3a3be37b8f72eba)